### PR TITLE
[3.11] gh-109634: Use :samp: role (GH-109635)

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
(cherry picked from commit 92af0cc580051fd1129c7a86af2cbadeb2aa36dc)

* Issue: gh-109634

----
:books: Documentation preview :books:: https://cpython-previews--109778.org.readthedocs.build/

### CODE REVIEW

File: Doc/library/codecs.rst [hunk 1]
```diff
@@ -1346,9 +1349,10 @@ encodings.
 |                    |         | supported.                |
 +--------------------+---------+---------------------------+
 | raw_unicode_escape |         | Latin-1 encoding with     |
-|                    |         | ``\uXXXX`` and            |
-|                    |         | ``\UXXXXXXXX`` for other  |
-|                    |         | code points. Existing     |
+|                    |         | :samp:`\\u{XXXX}` and     |
+|                    |         | :samp:`\\U{XXXXXXXX}``    |
```
Comment (JacobCoffee) [inline Doc/library/codecs.rst]:
```suggestion
|                    |         | :samp:`\\U{XXXXXXXX}`    |
```
An extra backtick here

File: Doc/reference/lexical_analysis.rst [hunk 2]
```diff
@@ -545,51 +545,51 @@ Unless an ``'r'`` or ``'R'`` prefix is present, escape sequences in string and
...
++-------------------------+---------------------------------+-------+
+| ``\n``                  | ASCII Linefeed (LF)             |       |
++-------------------------+---------------------------------+-------+
+| ``\r``                  | ASCII Carriage Return (CR)      |       |
++-------------------------+---------------------------------+-------+
+| ``\t``                  | ASCII Horizontal Tab (TAB)      |       |
++-------------------------+---------------------------------+-------+
+| ``\v``                  | ASCII Vertical Tab (VT)         |       |
++-------------------------+---------------------------------+-------+
+| :samp:`\\{ooo}`         | Character with octal value      | (2,4) |
```
Comment (JacobCoffee) [inline Doc/reference/lexical_analysis.rst]: Is this missing the `\`? - it just renders as `{ooo}` in the preview docs.
```suggestion
| :samp:`\{ooo}`         | Character with octal value      | (2,4) |
```

File: Misc/NEWS.d/3.8.0a1.rst [hunk 3]
```diff
@@ -8253,7 +8253,7 @@ Explain how IDLE's Shell displays output.
 
 Improve the doc about IDLE running user code.   The section is renamed from
 "IDLE -- console differences" is renamed "Running user code". It mostly
-covers the implications of using custom sys.stdxxx objects.
+covers the implications of using custom :samp:sys.std{xxx}` objects.
```
Comment (JacobCoffee) [inline Misc/NEWS.d/3.8.0a1.rst]:
```suggestion
covers the implications of using custom :samp:`sys.std{xxx}` objects.
```

**Review by JacobCoffee — CHANGES_REQUESTED** [review]
Found a couple of oopsies

File: Doc/reference/lexical_analysis.rst (hunk 2 above)
Comment (JacobCoffee) [reply Doc/reference/lexical_analysis.rst]: Prod:
image
Preview:
image

**Comment by serhiy-storchaka** [comment]
:man_facepalming: Good catch @JacobCoffee. Do you mind to create a PR for main branch?

**Comment by JacobCoffee** [comment]
Would this be based off of this branch or just with the fixes found?

File: Doc/library/codecs.rst (hunk 1 above)
Comment (serhiy-storchaka) [inline Doc/library/codecs.rst]: It should be replaced with a space, not removed, to preserve the table formatting.
```suggestion
|                    |         | :samp:`\\U{XXXXXXXX}`     |
```

File: Doc/reference/lexical_analysis.rst (hunk 2 above)
Comment (serhiy-storchaka) [inline Doc/reference/lexical_analysis.rst]: No, it needs more backslashes, not less.
```suggestion
| :samp:`\\\\{ooo}`       | Character with octal value      | (2,4) |
```

**Comment by serhiy-storchaka** [comment]
Since the original changes were already merged in main and 3.12, they only need fixes. Note my corrections of two of your suggestions.

**Comment by JacobCoffee** [comment]
#110073 is ready for you @serhiy-storchaka

**Review by JacobCoffee — APPROVED** [review]

**Pull request merged**